lib/rmg/ObjectManager.h

```diff
@@ -53,10 +53,18 @@ class ObjectManager: public Modificator
 	bool createRequiredObjects();
 
 	int3 findPlaceForObject(const rmg::Area & searchArea, rmg::Object & obj, si32 min_dist, OptimizeType optimizer) const;
-	int3 findPlaceForObject(const rmg::Area & searchArea, rmg::Object & obj, std::function<float(const int3)> weightFunction, OptimizeType optimizer) const;
+	int3 findPlaceForObject(const rmg::Area & searchArea,
```

Comment:
Questionable formatting. 
@kambala-decapitator any opinion on how should we format declarations in headers?

Reply:
reverted

Reply:
But we need to establish rule for long declarations in header.